Add azure page (#200)
GlareDB/glaredb.github.io: docs/data-sources/supported/azure.md

---
layout: default
title: Azure Blob Storage
parent: Supported data sources
grand_parent: "Step 1: Connect your data sources"
---

<!-- markdownlint-disable MD022 -->

<!-- prettier-ignore-start -->
# Azure Blob Storage
{: .no_toc }
<!-- prettier-ignore-end -->

<!-- markdownlint-enable MD022 -->

> CSV and Parquet files in Azure Blob Storage are able to be used as an external
> data source.

Reviewer: ```suggestion
GlareDB can use CSV and Parquet files in Azure Blob Storage as an external
data source.
```

<!-- prettier-ignore-start -->

- TOC
{:toc}
<!-- prettier-ignore-end -->

## Azure file as an external table

> Add CSV and Parquet files as an external table with the

Reviewer: ```suggestion
Add CSV, ndjson, or Parquet files as an external table with the
```

> CSV and Parquet files can be added as an external table through the
> [CREATE EXTERNAL TABLE] command.

Reviewer: ```suggestion
Add CSV and Parquet files as an external table with the
[CREATE EXTERNAL TABLE] command.
```

{: .important}

> > `table-name` will be the name of the database inside GlareDB. `table-name` may
> > optionally be qualified with a schema name.

Reviewer: ```suggestion
> `table-name` becomes the name of the database inside GlareDB. `table-name` can 
> be qualified with a schema name.
```
